packages/assets-controllers/src/NftController.ts

```diff
@@ -512,6 +512,53 @@ export class NftController extends BaseController<
     });
   }
 
+  /**
+   * Fetches NFT Collection Metadata from the NFT API.
+   *
+   * @param contractAddresses - The contract addresses of the NFTs.
+   * @param chainId - The chain ID of the network where the NFT is located.
+   * @returns NFT collections metadata.
+   */
+  async fetchNftCollectionsMetadata(
```

Comment:
Minor, but does this controller also try and keep public methods at the top for visibility?

Reply:
I was also looking at this but there is no strong convention that this controller follows, the last two private method is breaking the rule. 
Also publics are used at bottom here, not at the top, so it's a bit opposite as we do 🙂
Anyways, I relocated this public method with others.